package.json

```diff
@@ -46,5 +46,8 @@
     "jest-ts-webcompat-resolver": "1.0.0",
     "prettier": "2.3.2",
     "typescript": "4.3.5"
+  },
+  "dependencies": {
+    "tslib": "^2"
```

Comment:
What makes this project depend on tslib? I've constantly found myself struggling to understand when I need to add tslib as a dependency vs when I don't/

Reply:
https://github.com/foxglove/template-typescript/issues/153

Reply:
You must depend on tslib when importHelpers is set to true. Ideally, it is set to true in all TypeScript libraries your project depends on so TS standard library functions are compiled into your app once instead of repeated in the compiled source of each package.